<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetImporterWindow</class>
 <widget class="QDialog" name="AssetImporterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AssetImporter Qt</string>
  </property>
  <layout class="QGridLayout" name="mainLayout" columnstretch="0,0">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="column1">
     <item>
      <widget class="QPushButton" name="reloadLibButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh Library</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="libraryFrame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scroll">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>369</width>
          <height>539</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="libraryLayout">
         <item>
          <widget class="QPushButton" name="deleteMe">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="column2">
     <item>
      <widget class="QPushButton" name="reloadImportedButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh Imported</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="importedList"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
